package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "vuepress": "^0.6.0"
+  }
+}
```

Comment:
what about moving this into `/guide`? I just don't want to confuse devs that see this `package.json` in the root and maybe expect to be able to build something useful for solidus development? I think that the yarn commands can be run into that folder directly as well, right?

Reply:
This was my first take, but something led me to move this into the root folder. I will try again, because I agree that this should not be in the root folder.